### PR TITLE
build(docs-infra): update to cli@9.0.0-next

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Comment by IgorMinar** [comment]
@gkalpak I marked this as master-only, but I can be convinced of pushing this to prod as well if it makes future cherry-picking easier.

File: aio/package.json [hunk 1]
```diff
@@ -84,28 +84,28 @@
...
+    "@angular/platform-browser": "^9.0.0-next.6",
+    "@angular/platform-browser-dynamic": "^9.0.0-next.6",
+    "@angular/router": "^9.0.0-next.6",
+    "@angular/service-worker": "^9.0.0-next.6",
     "@types/lunr": "^2.3.2",
     "@webcomponents/custom-elements": "^1.2.0",
-    "rxjs": "^6.5.2",
-    "zone.js": "^0.10.2"
+    "rxjs": "^6.5.3",
+    "zone.js": "~0.10.1"
```
Comment (gkalpak) [inline aio/package.json]: Why this change? Was it done by the cli?

**Review by gkalpak — APPROVED** [review]
IIRC, the latest cli version will only build from es2015 (and downlevel to es5 if necessary). If that is the case, we could probably change [scripts/switch-to-ivy.js][1] to only process es2015 with ngcc.
(That should speed the corresponding CI job a bit, but should not block this PR.)

```diff
-const ngccArgs = '--loglevel debug --properties es2015 module';
+const ngccArgs = '--loglevel debug --properties es2015';
```

[1]: https://github.com/angular/angular/blob/354f99016/aio/scripts/switch-to-ivy.js#L39

**Comment by gkalpak** [comment]
> @gkalpak I marked this as master-only, but I can be convinced of pushing this to prod as well if it makes future cherry-picking easier.

I generally prefer keeping `next` and `stable` in sync wrt these dependencies (and that's what we've been doing iirc), but not a hill I am going to die on (esp. since I don't it is very likely that we'll need to make changes to `stable` that would cause conflicts).

File: aio/package.json (hunk 1 above)
Comment (IgorMinar) [reply aio/package.json]: yes

File: aio/angular.json [hunk 2]
```diff
@@ -84,6 +102,12 @@
               "serviceWorker": true
             },
             "archive": {
+              "budgets": [
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb"
+                }
+              ],
```
[the remark this answers is not in the document]
Comment (IgorMinar) [reply aio/angular.json]: this was the cli default. I can change it.

[the remark this answers is not in the document]
Comment (IgorMinar) [reply aio/angular.json]: done

File: aio/package.json (hunk 1 above)
Comment (IgorMinar) [reply aio/package.json]: yes

**Comment by gkalpak** [comment]
As discussed offline, we need to verfy whether this turns on ivy for angular.io and ensure we stilk test and track payload size against ViewEngine.

**Comment by gkalpak** [comment]
Actually, I hadn't realized that since ec4381dd4 we have `enableIvy: false` in our config. So, this PR does not turn on Ivy.

I suggest we merge this one and I will send another PR to turn Ivy on for angular.io (and ensure we keep testing against ViewEngine).

**Review by Splaktar — APPROVED** [review]
LGTM

**Comment by atscott** [comment]
Hey @IgorMinar - this merged to master but there was a merge conflict with the patch branch. Can you create a new PR targeting patch only? Thanks!

**Comment by atscott** [comment]
Edit: Sorry, it didn't merge to master. I misread the script message.

**Comment by atscott** [comment]
Okay, *now* it's merged to master. @IgorMinar, please create a new PR targeting the patch branch.
